script/setup/install-dev-tools

```diff
@@ -27,6 +27,8 @@ go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.54.2
 # the following packages need to exist in $GOPATH so we can't use
 # go modules-aware mode of `go get` for these includes used during
 # proto building
-cd "$GOPATH"
-GO111MODULE=off go get -d github.com/gogo/googleapis || true
-GO111MODULE=off go get -d github.com/gogo/protobuf || true
+GOGO_DIR="$GOPATH/src/github.com/gogo"
```

Comment:
/cc @dmcgowan 

This change is done so that we can get the gogo proto sources when using 1.22 as the default go get behaviour has changed.